src/analyze/analyze-security.c

```diff
@@ -2553,7 +2553,7 @@ static int get_security_info(Unit *u, ExecContext *c, CGroupContext *g, Security
                                 info->restrict_address_family_other = !c->address_families_allow_list;
                 }
 
-                info->restrict_namespaces = c->restrict_namespaces;
+                info->restrict_namespaces = c->retain_namespaces;
```

Comment:
Why not rename the lhs too?

Reply:
Because it maps to an external field. This commit is purely about changing non user visible interfaces. The original author of the feature can do the work to clean up the user visible interfaces ;)

Reply:
Actually I guess we can address this field as well, fixed